stdlib/seq.mli

```diff
@@ -210,6 +210,17 @@ val exists : ('a -> bool) -> 'a t -> bool
 
     @since 4.14 *)
 
+val mem : 'a -> 'a t -> bool
+(** [mem x xs] is true if and only if [x] is equal to an element of [xs].
+
+    @since 5.4 *)
```

Comment:
```suggestion
(** [mem x xs] is true if and only if [x] is equal to an element of [xs].

    The sequence [xs] must be finite.

    @since 5.4 *)
```

Reply:
I know the module is littered with "`The sequence [xs] must be finite.`" (e.g. `Seq.find`) but most of these statements look wrong to me. I think a more correct statement would be: 

`Guaranteed to terminate only if [xs] is finite.`

Reply:
I agree, and a PR making the comments more precise, as suggested, would be welcome.
However, as far as the current PR is concerned, I still think it should follow whatever the current style is.

Reply:
That's not style, that's wrong.